packages/components/src/date-time/index.js

```diff
@@ -1,10 +1,6 @@
 /**
  * External dependencies
  */
```

Comment:
Since there are no longer any external dependencies imported here, this comment can be removed.